target/linux/ramips/dts/mt7620n_hootoo_tm05.dts

```diff
@@ -0,0 +1,129 @@
+/dts-v1/;
+
+#include "mt7620n.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "hootoo,tm05", "ralink,mt7620n-soc";
+	model = "HooToo TM05";
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		green {
+			label = "tm05:green";
+			gpios = <&gpio2 0 GPIO_ACTIVE_HIGH>;
+			default-state = "on";
+			// Enabled below line, if flashing LED desired
+			//linux,default-trigger = "timer";
+		};
+
+		blue {
+			label = "tm05:blue";
+			gpios = <&gpio3 0 GPIO_ACTIVE_LOW>;
+			default-state = "on";
```

Comment:
Why?

Reply:
Sorry, why what? If you mean the default state - I found that other devices (like the one I copied from) have the active high/low logic inverted (and actually incorrect) ... but they do it so that with a default state of off, the LED turns on. I have the same end result (LED on), but with correct logic polarity, and setting the default to on (i.e. so the LED turns on when up and running).

Reply:
I just wonder why you have two LEDs that are supposed to be default-on without a clear indication of what they are used for. Having read your commit message, you should probably change them to:
- "power" or "status" for the blue one
- "wifi" for the green one

Reply:
Sure, will do (changing to power and wifi, like you suggest).

Reply:
Since you have a power LED now, you can use it to indicate boot, failsafe, etc. Just have a look at the DTS files of other devices or grep for "led-boot", "led-failsafe", etc.

Reply:
Updated - and you're right, very cool. Now I see different LED signatures, as it boots, etc. Thanks!

Reply:
This seems to be as noted above - will mark as resolved, but yell if I'm wrong!